gnomad_qc/v4/create_release/create_release_sites_ht_exomes.py

```diff
@@ -658,9 +658,9 @@ def main(args):
         args.release_exists,
     )
 
-    # Filter out chrM and AS_lowqual sites.
+    # Filter out chrM, AS_lowqual sites, and AC_raw == 0.
     ht = hl.filter_intervals(ht, [hl.parse_locus_interval("chrM")], keep=False)
-    ht = ht.filter(hl.is_defined(ht.filters))
+    ht = ht.filter(hl.is_defined(ht.filters) & ht.freq[1].AC > 0)
```

Comment:
Shouldn't it be `ht = ht.filter(hl.len(ht.filters)==0) & (ht.freq[1].AC > 0))`? Also `AS_lowqual` seems to be from get_info() and we didn't select, no? `filters` is from `final_filter().ht()`, right?

Reply:
oh okay, but you need () for (ht.freq[1].AC > 0), otherwise there's an TypeError.

Reply:
Yes thanks thats been updated